composeApp/src/commonMain/kotlin/world/respect/app/view/apps/list/AppListScreen.kt

```diff
@@ -0,0 +1,95 @@
+package world.respect.app.view.apps.list
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Android
+import androidx.compose.material.icons.filled.Link
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.navigation.NavHostController
+import com.ustadmobile.libuicompose.theme.black
+import com.ustadmobile.libuicompose.theme.white
+import org.jetbrains.compose.resources.stringResource
+import respect.composeapp.generated.resources.Res
+import respect.composeapp.generated.resources.add_link
+import respect.composeapp.generated.resources.add_from_link
+import world.respect.app.app.EnterLink
+import world.respect.app.appstate.getTitle
+import world.respect.app.viewmodel.apps.list.AppListViewModel
+
+
+@Composable
+fun AppListScreen(
+    navController: NavHostController,
+    viewModel: AppListViewModel
+) {
+
+    val uiState by viewModel.uiState.collectAsState()
+
```

Comment:
Everythign should be in the LazyColumn. Here add a link will get stuck at the top and won't scroll. Add a link as per the design is just part of the list.